### DISCUSSION TITLE
Ajuda com extensão Git para visualizar arquivos do pacote Office

### DISCUSSION THREAD

**Discussion by leticiadelfino** [frontendbr/forum#1153] [Perguntas e Respostas]
Pessoal, trabalho com análise de requisitos e precisava versionar documentos do pacote Office pelo Git. Mas quando adiciono um documento .docx ou .xlsx o Git não consegue abrir para visualização. Alguém sabe de alguma extensão que me permita visualizar esses arquivos sem ter que baixar e abrir pelo pacote Office? Tentei usar o pandoc, mas ele não conseguiu reconhecer as imagens e tabelas do documento.

**Comment by danjesus** [comment]
@leticiadelfino não entendi muito bem, está falando de abrir no github mesmo online?

**Comment by leticiadelfino** [comment]
Sim @danjesus . Queria ver se existe alguma extensão que me permita abrir esses arquivos no Git sem precisar baixar para ver o que foi alterado.

**Comment by renanlara** [comment]
Use Google Docs xD
Ele faz o versionamento de todas as alterações.

**Comment by danjesus** [comment]
@leticiadelfino dentro do github isso não é possível, usando o Google docs como o @renanlara mencionou tudo já é versionado por padrão.

Localmente você consegue abrir no seu computador se inserir o git no fluxo de trabalho das pessoas, mesmo assim as mudanças não seriam muito atômicas.

No Google docs cada mudança é armazenada e pode ser restaurada, além de exibir quem mudou de maneira muito mais simples e permitir o trabalho colaborativo, com muitas pessoas editando o mesmo arquivo.
